aws/resource_aws_rds_cluster_test.go

```diff
@@ -1651,6 +1677,54 @@ resource "aws_db_subnet_group" "test" {
 `, bucketName, bucketPrefix, uniqueId, uniqueId, uniqueId, bucketPrefix, uniqueId, uniqueId, uniqueId, uniqueId)
 }
 
+func testAccAWSClusterConfig_pointInTimeRestoreSource(parentId, childId string) string {
+	return fmt.Sprintf(`
+data "aws_availability_zones" "available" {}
+
+resource "aws_rds_cluster" "test" {
+  cluster_identifier = "%s"
+  master_username = "root"
+  master_password = "password"
+  db_subnet_group_name = "${aws_db_subnet_group.test.name}"
+  skip_final_snapshot = true
+  engine = "aurora-mysql"
+}
+
+resource "aws_vpc" "test" {
+  cidr_block = "10.0.0.0/16"
+  tags = {
+    Name = "%s-vpc"
+  }
+}
+
+resource "aws_subnet" "subnets" {
+  count = "${length(data.aws_availability_zones.available.names)}"
+  vpc_id = "${aws_vpc.test.id}"
+  cidr_block = "10.0.${count.index}.0/24"
+  availability_zone = "${data.aws_availability_zones.available.names[count.index]}"
+  tags = {
+    Name = "%s-subnet-${count.index}"
+  }
+}
+
+resource "aws_db_subnet_group" "test" {
+  name = "%s-db-subnet-group"
+  subnet_ids = ["${aws_subnet.subnets.*.id}"]
+}
+
+resource "aws_rds_cluster" "restored_pit" {
+	cluster_identifier = "%s"
+	skip_final_snapshot = true
+	engine = "aurora-mysql"
```

Comment:
Nit: can reference the other resource here 👍 
```suggestion
	engine = aws_rds_cluster.test.engine
```